functions.js

```diff
@@ -0,0 +1,23 @@
+const navbar = document.querySelector('.navbar');
+const openButton = document.querySelector('.hamb-icon');
+const closeButton = document.querySelector('.close-btn');
+
+openButton.addEventListener('click', () => {
+  navbar.classList.toggle('active');
+});
+
+closeButton.addEventListener('click', () => {
+  navbar.classList.toggle('active');
+});
+
+// var mediaquery = window.matchMedia('(min-width: 769px');
+// let actButton = document.querySelector('.act-btn');
+// if(mediaquery.matches) {
+//   actButton.addEventListener('click', () => {
+//     console.log('clickeando' , mediaquery);
+//   })
+// }else{
+//   actButton.addEventListener('click', () => {
+//     console.log('chico' , mediaquery);
+//   })
+// }
```

Comment:
- [ ]  Please remove these comments from your code because comments are typically used to explain code logic or temporarily disable certain code sections. However, leaving commented code in the codebase can lead to maintenance issues. Over time, it can become outdated, irrelevant, or even contradict the actual functioning code. This can make debugging and troubleshooting more challenging, as developers may waste time analyzing code that is no longer active or valid.